gsutil.py

```diff
@@ -84,6 +84,8 @@ def OutputAndExit(message):
     ('httplib2', submodule_pyvers),
     ('retry-decorator', ''),
     ('six', ''),  # Python 2 / 3 compatibility dependency
+    ('google-auth-library-python', ''),
+    ('cachetools', ''),
```

Comment:
I think this line should be before google-auth as it is the dependency for googl-auth. Please also leave a comment saying that this is for google-auth dependency.

Reply:
Done

Reply:
Done